.github/workflows/cicd.yml

```diff
@@ -96,8 +96,8 @@ jobs:
           cmake ../EmptyEpsilon
           -G "Visual Studio 16 2019" -A Win32
           -DCMAKE_BUILD_TYPE=${{ matrix.build_type }}
-          -DSFML_ROOT="${{github.workspace}}/externals/SFML-2.5.1"
-          -DSERIOUS_PROTON_DIR=../SeriousProton
+          -DSFML_ROOT="../externals/SFML-2.5.1"
```

Comment:
packaging fails if using the full paths (because the `github.workspace` value contains backslashes)